ansible_runner/utils/streaming.py

```diff
@@ -64,5 +72,11 @@ def unstream_dir(stream, length, target_directory):
             for info in archive.infolist():
                 archive.extract(info.filename, path=target_directory)
                 out_path = os.path.join(target_directory, info.filename)
-                perm = info.external_attr >> 16
-                os.chmod(out_path, perm)
+                perms = info.external_attr >> 16
+                mode = stat.filemode(perms)
+                if mode[:1] == 'l':
+                    link = open(out_path).read()
+                    os.remove(out_path)
+                    os.symlink(link, out_path)
```

Comment:
We want to retain the perms from the original project, even on the symlink.

Reply:
I think this is just how symlinks work:

```
$ cd /tmp/
$ ln -sf /etc/passwd /tmp/hello
$ sudo su - anotheruser
Last login: Thu Jun  3 15:34:33 EDT 2021 on pts/27
$ rm /tmp/hello
rm: cannot remove '/tmp/hello': Operation not permitted
```